js/datepicker.js

```diff
@@ -604,11 +602,15 @@
         j = 1 + year + opts.yearRange;
       }
 
+      let years = [];
       for (arr = []; i < j && i <= opts.maxYear; i++) {
         if (i >= opts.minYear) {
-          arr.push(`<option value="${i}" ${i === year ? 'selected="selected"' : ''}>${i}</option>`);
+          years.push(
+            `<option value="${i}" ${i === year ? 'selected="selected"' : ''}>${i}</option>`
+          );
         }
       }
+      arr = arr.concat(opts.yearRangeReverse ? years.reverse() : years);
```

Comment:
Not sure why arr.concat is used here. You can probably leave the old code untouched and use one line with this:

```
if(opts.yearRangeReverse) {
arr.reverse();
}
```

Which will reverse the array elements in place.

Reply:
You might also want to add it as default at the top.

Reply:
@DanielRuf I fixed the overhead.
Pls. what do you mean by `add it as default at the top` ?

Reply:
See https://github.com/Dogfalo/materialize/blob/v1-dev/js/datepicker.js#L4, the defaults are readable and visible when you output a component instance on the console.